Example/ViewController.swift

```diff
@@ -146,12 +146,15 @@ final class ViewController: UIViewController {
         guard let authStateManager = authStateManager else { return }
         
         oktaAppAuth?.signOut(authStateManager: authStateManager, from: self, progressHandler: { currentOption in
-            if currentOption.contains(.revokeAccessToken) {
+            switch currentOption {
+            case .revokeAccessToken, .revokeRefreshToken, .removeTokensFromStorage, .revokeTokensOptions:
                 self.updateUI(updateText: "Revoking tokens...")
-            } else if currentOption.contains(.revokeRefreshToken) {
-                self.updateUI(updateText: "Revoking tokens...")
-            } else if currentOption.contains(.signOutFromOkta) {
+            case .signOutFromOkta:
                 self.updateUI(updateText: "Signing out from Okta...")
+            case .allOptions:
+                break
```

Comment:
Why do we need to handle `.allOptions` case? The behaviour is the same as for `default`

Reply:
Not really, if a new case will be added to enum, the compiler suggests to fulfill `switch case`. 

Also, it's a good practice to make switch exhaustive to show cases which are not handled.

Reply:
Done)